File: weather_script.js
const fetch = require('node-fetch');

const apiKey = 'c7c51bb77d9018aa3141d03ab118a940';
const city = 'London';
const url = `https://api.openweathermap.org/data/2.5/weather?q=${city}&appid=${apiKey}`;

fetch(url)c7c51bb77d9018aa3141d03ab118a940

    .then(response => response.json())
    .then(data => {
        console.log(`Weather in ${city}:`);
        console.log(`Temperature: ${data.main.temp}K`);
        console.log(`Weather: ${data.weather[0].description}`);
    })
    .catch(error => console.error('Error fetching weather data:', error));

File: weather_script.py
import requests

def get_weather(api_key, city):
    base_url = "http://api.openweathermap.org/data/2.5/weather"
    params = {
        'q': city,
        'appid': api_key,
        'units': 'metric'
    }
    response = requests.get(base_url, params=params)
    if response.status_code == 200:
        data = response.json()
        main = data['main']
        weather = data['weather'][0]
        temperature = main['temp']
        humidity = main['humidity']
        weather_description = weather['description']
        print(f"Temperature: {temperature}°C")
        print(f"Humidity: {humidity}%")
        print(f"Weather Condition: {weather_description}")
    else:
        print("Error in the HTTP request")

if __name__ == "__main__":
    api_key = "your_api_key_here"
    city = input("Enter city name: ")
    get_weather(api_key, city)